src/Java.Interop/Java.Interop/JniRuntime.cs

```diff
@@ -291,7 +290,7 @@ static unsafe JavaVMInterface CreateInvoker (IntPtr handle)
 		public virtual void FailFast (string? message)
 		{
 			var m = typeof (Environment).GetMethod ("FailFast");
-			m.Invoke (null, new object?[]{ message });
+			m?.Invoke (null, new object?[]{ message });
```

Comment:
This is a "FailFast" invocation.  If `m` is null, for whatever reason, it *should* throw.

*Or*, we should add a `throw new ExecutionEnginException()` or something on the following line.

The intent here is "the VM is hosed; we need to die"